packages/orm/lib/orm.js

```diff
@@ -66,13 +66,22 @@ const runPaginatedQuery = async ({ pageSize = 1, variables = {}, verbose = false
   return allItems;
 };
 
-const runChunkedMutation = async ({ pageSize = 500, items, ...rest }) => {
+const runChunkedMutation = async ({ listName, pageSize = 500, items, ...rest }) => {
   const chunks = chunkArray(items, pageSize);
 
+  const { createManyMutationName } = rest.keystone.lists[listName].gqlNames;
```

Comment:
Pulling out `createManyMutationName` here will only be the right thing to do if the chunked mutation is a `create` operation. We should really be passing in the `gqlName` from whoever calls `runChunkedMutation`.

Reply:
Ah, that's a very good point. Thanks for bringing it up. @timleslie

Reply:
This is a great idea - createItems should pass that 👍